apps/explorer/src/pages/checkpoints/CheckpointsTable.tsx

```diff
@@ -0,0 +1,170 @@
+    );
+
+    const checkpointsTable = useMemo(
+        () =>
+            data
+                ? {
+                      data: data?.data.map((checkpoint) => ({
+                          digest: (
+                              <TxTableCol isHighlightedOnHover>
+                                  <CheckpointLink digest={checkpoint.digest} />
+                              </TxTableCol>
+                          ),
+                          time: (
+                              <TxTableCol>
+                                  <TxTimeType
+                                      timestamp={checkpoint.timestampMs}
+                                  />
+                              </TxTableCol>
+                          ),
+                          sequenceNumber: (
+                              <TxTableCol>
+                                  <Text
+                                      variant="bodySmall/medium"
+                                      color="steel-darker"
+                                  >
+                                      {checkpoint.sequenceNumber}
+                                  </Text>
+                              </TxTableCol>
+                          ),
+                          transactionCount: (
+                              <TxTableCol>
+                                  <Text
+                                      variant="bodySmall/medium"
+                                      color="steel-darker"
+                                  >
+                                      {checkpoint.transactions.length}
+                                  </Text>
+                              </TxTableCol>
+                          ),
+                      })),
+                      columns: [
+                          {
+                              header: 'Digest',
+                              accessorKey: 'digest',
+                          },
+                          {
+                              header: 'Sequence Number',
+                              accessorKey: 'sequenceNumber',
+                          },
+                          {
+                              header: 'Time',
+                              accessorKey: 'time',
+                          },
+                          {
+                              header: 'Transaction Count',
+                              accessorKey: 'transactionCount',
+                          },
+                      ],
+                  }
+                : null,
+        [data]
+    );
+
+    const stats = {
+        count: countQuery.data || 0,
+        stats_text: 'Total Checkpoints',
+        loadState: statusToLoadState[countQuery.status],
+    };
+
+    return checkpointsTable ? (
+        <>
+            <TableCard
+                data={checkpointsTable.data}
+                columns={checkpointsTable.columns}
+            />
+            {handlePageChange &&
+            paginationType !== 'none' &&
```

Comment:
can we delete `paginationType !== 'none'` since we're explicitly checking `paginationType === 'pagination'`?